### DISCUSSION TITLE
Laravel Reverb + bref

### DISCUSSION THREAD

**Discussion by cirolosapio** [brefphp/bref#1774] [Support]
Is possible to run Laravel Reverb with bref ?

**Comment by mnapoli** [comment]
You'd need to run it on a server. But you could use API Gateway Websocket for a serverless solution, that's what I use in my projects.

**Reply by ttsuru** [reply]
@mnapoli

Is it possible to run Laravel's Broadcasting compatibility with AWS API Gateway + Lambda?
Or any ideas?

**Reply by ttsuru** [reply]
I am looking at the following projects
https://github.com/georgeboot/laravel-echo-api-gateway
https://soketi.app/

**Reply by emaadali** [reply]
I use [georgeboot/laravel-echo-api-gateway](https://github.com/georgeboot/laravel-echo-api-gateway) without any issues

**Reply by mnapoli** [reply]
On my end I've implemented an integration between API Gateway and the Broadcasting component of Laravel. It isn't released at the moment though.

**Reply by ttsuru** [reply]
@mnapoli 
Do you have any plans for a release?

**Reply by mnapoli** [reply]
Not in my immediate plans, I need to finish the X-Ray integration first.

**Reply by dropout-ventures** [reply]
@mnapoli is there an update on this? we are trying to switch to Reverb, and would like to continue using API Gateway.

is there a code example or tutorial on how you did your implementation?

**Reply by mnapoli** [reply]
@dropout-ventures hey! unfortunately I haven't published anything at the moment!

**Reply by dropout-ventures** [reply]
> @dropout-ventures hey! unfortunately I haven't published anything at the moment!

anything would be helpful 😄

**Reply by emaadali** [reply]
@dropout-ventures I use https://github.com/georgeboot/laravel-echo-api-gateway to handle hundreds of thousands of messages per day. I haven't run into any issues and it wasn't hard to set up. The repo hasn't been updated in a long time so it is probably abandoned, but it might be something you can use in the meantime while you wait for @mnapoli

**Reply by dropout-ventures** [reply]
@emaadali yes that's what I am currently using, but it's deprecated so would like to upgrade if possible

**Reply by emaadali** [reply]
I've been meaning to make a replacement for compliance reasons, so maybe I'll do that this weekend.

**Comment by hlspablo** [comment]
someone managed to make soketi works with bref ?

**Comment by JGamboa** [comment]
@mnapoli  have you done any publication of the code you using?

**Reply by mnapoli** [reply]
@JGamboa I still don't unfortunately, I haven't packaged it properly.
